apps/pay/app/lnurlp/[username]/callback/route.ts

```diff
@@ -114,9 +116,13 @@ export async function GET(
     })
   }
 
+  // eslint-disable-next-line
+  // @ts-ignore
+  const url = originalUrl(request)
```

Comment:
We already use this function `originalUrl` elsewhere in the project, however the types do not exactly match up. It is used to get the original url in the case that the request was proxied. A follow up would be to migrate the logic from originalUrl to this repo so that the types can be correct.

Reply:
I will actually just migrate the logic now.